skiko/src/commonMain/kotlin/org/jetbrains/skia/Font.kt

```diff
@@ -148,16 +148,25 @@ class Font : Managed {
         }
 
     /**
-     * @return  true if font and glyph metrics are requested to be linearly scalable
+     * Returns true if font and glyph metrics are requested to be linearly scalable.
+     *
+     * @return  true if font and glyph metrics are requested to be linearly scalable.
      */
-    fun areMetricsLinear(): Boolean {
-        return try {
+    var isMetricsLinear: Boolean
```

Comment:
```suggestion
    var isLinearMetrics: Boolean
```

I would name it this way, according the old [Skija policy](https://github.com/JetBrains/skija/blob/master/docs/Conventions.md).